processor/fanoutconnector_test.go

```diff
@@ -31,7 +31,7 @@ func TestTraceProcessorMultiplexing(t *testing.T) {
 		processors[i] = &mockTraceConsumer{}
 	}
 
-	tdp := NewTraceProcessor(processors)
+	tdp := NewTraceFanOutConnector(processors)
```

Comment:
not from your change but I see that `tdp` also is not a good name...

Reply:
I agree, fixing.

Reply:
Fixed.

Reply:
This one is still `tdp`

Reply:
Sorry, missed in the tests. Fixed now.